### ISSUE TITLE
Request for server API script without sessions 

### ISSUE THREAD

**Issue by CORRUPTOR2037** [turboderp/exllama#87]
So I could just send a simple request and get simple response in a free-form mode, without any additional context

**Comment by turboderp** [comment]
This is pretty much what the `example_flask.py` script does. Is that what you're after?

**Comment by CORRUPTOR2037** [comment]
Oh god really, it is, I'm sorry
But it will be even better if I could pass generation settings with inference post request or set them in a separate request. I know it's not a big deal, I can rewrite it myself, but it will be so much better for everyone if such script will be available from the box.

**Comment by CORRUPTOR2037** [comment]
Thanks. Jumped here from oobabooga's, I really like your minimal dependency approach in contrast with enourmous blob of hundred dependencies mixed with conda, keep it up!

**Comment by BlankParenthesis** [comment]
I posted it in another issue at some point, but I have a script which sounds like what you want here: https://gist.github.com/BlankParenthesis/4f490630b6307ec441364ab64f3ce900

It mimics ooba's text-generation-webui API so it works with tavern (the main difference being the streaming API is the same port as the regular API).

I tried to keep it simple, but I ended up having to make some changes for it to work nicely. Some of those workarounds make the generate function a bit more complex than I'd like so perhaps there's a nice way to do it, but this seems to work at least. The settings mismatch is also a bit ugly: by following the existing format it excludes exllama specific settings and ignores others.

There's no way to set the model using this, but adding a route for that should be simple enough if you want.
